coda/coda_validate/urls.py

```diff
@@ -6,6 +6,7 @@
     url(r'^APP/validate/(?P<identifier>.+?)/$', views.app_validate, name='app-detail'),
     url(r'^APP/validate/$', views.app_validate, name='app-list'),
     url(r'^validate/$', views.index, name='index'),
+    url(r'^validate/check.json/$', views.check_json, name='check_json'),
```

Comment:
I am not sure if there should be a slash or not. I personally think that there should be no trailing slashes on any of the URLs, but its not my decision. Tell me what you and @vphill want, and I will make it so :picard_meme:

:smile_cat: